snippets/stripe-webhook.md

```diff
@@ -0,0 +1,159 @@
+# Stripe Webhook Integration  
+
+This introduces how to create an API with Hono to receive Stripe Webhook events.
+
+## Preparation
+
+Please install the official Stripe SDK at first:
+
+```bash
+% npm install stripe
```

Comment:
Please remove `% ` to make it easy to copy the line.

Reply:
@yusukebe removed 👍